generate-github-app-token-from-vault-secrets/README.md

```diff
@@ -18,6 +18,8 @@ This composite GHA can be used in any repository.
 | vault-auth-secret-id                | The Secret Id for (Vault) App Role authentication                       |
 | vault-url                           | The URL for the Vault endpoint                                          |
 | skip-token-revoke                   | If truthy, the token will not be revoked when the current job is complete (optional) |
+| owner                               | The owner of the GitHub App installation (defaults to current repository owner, optional). |
+| repositories                        | Comma or newline-separated list of repositories to install the GitHub App on (defaults to current repository if owner is unset, optional). If you want to generate a token that has access to all repositories of the owner, set this to `!all` and explicitely set an `owner`. |
```

Comment:
since the app is not installed by this action, only a token obtained:
```suggestion
| repositories                        | Comma or newline-separated list of repositories for which the GitHub app token will be valid for(defaults to current repository if owner is unset, optional). If you want to generate a token that has access to all repositories of the owner, set this to `!all` and explicitely set an `owner`. |
```